_posts/2020-01-16-cycle-8-results.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Bisq DAO Cycle 8: Results"
+author: Steve Jain
+excerpt: "Cycle 8 of the Bisq DAO ended at block 609 186 on December 21 2019. This post covers its results. <br><br>"
+en-only: true
+---
+
+This post summarizes the results of Cycle 8 of the Bisq DAO.
+
+It comes a bit later than usual, as Cycle 8 wasn't a pretty one. During the cycle, some significant issues were made apparent in the human side of the network: a lack of management and budgeting resulted in suboptimal resource allotment needed for growth, which resulted in a ballooning BSQ supply, which resulted in an erosion of confidence in BSQ, which ultimately resulted in a sharp downturn in the BSQ price.
+
+In response, key contributors are working on substantial changes to the way Bisq operates in order to address these issues and restore confidence in the Bisq DAO. 
+
+These changes will be discussed more thoroughly in a call next week, as well as in a post on this blog shortly thereafter.
+
+### Summary
+
+* Cycle took place between blocks 604 507 and 609 186
+  * Calendar dates: 11/19/2019 - 12/21/2019
+* 37 proposals
+  * 1 generic proposal
+  * 1 parameter change and 1 bonded role
+  * 35 compensation requests
+* 445 votes cast
+* 124 083 BSQ issued for compensation (151 293 BSQ was issued in total, of which 27 210 was for refund agent)
+* 13 449 BSQ burned
+
+High BSQ issuance in the face of anemic trading volume resulted in a relatively big increase in BSQ supply during the cycle, a problem that aggravated the BSQ market. 
+
+As of the end of Cycle 8, BSQ was trading at about 9200 satoshis/BSQ, which was about $0.66 at the time (BSQ had been holding in the $0.80-1.00 range for much of the year). Since then the price has dropped quite a bit more, with many trades in the 3000-6000 satoshis/BSQ range. The bitcoin-USD rate has jumped over 20% since Cycle 8 ended, which has somewhat dampened the effect of the nominal price drop, but the real drop in BSQ price is still severe.
```

Comment:
s/severe/significant/. I don't think that diminishes from the seriousness of the statement, while it avoids evoking too much emotion, panic, alarm.